osclib/accept_command.py

```diff
@@ -51,6 +84,23 @@ def reset_rebuild_data(self, project):
         content = ET.tostring(root)
         http_PUT(url + '?comment=accept+command+update', data=content)
 
+    def virtual_accept_to_delete(self, request_id, package):
+        self.api.add_review(request_id, by_group=self.api.delreq_monitor)
+
+        pkgs = []
+        filelist = self.api.get_filelist_for_package(pkgname=package, project=self.api.project, expand='1', extension='spec')
+        for spec in filelist:
+            pkgs.append(spec[:-5])
```

Comment:
Another need for having a method to return list of subpackages based on spec files as noted above.